### PR TITLE
Improve flow of Custom-Voice-Controller example

### PR DESCRIPTION
**Pull request by S2Ler**

### CODE REVIEW

File: Navigation-Examples/Examples/Custom-Voice-Controller.swift [hunk 1]
```diff
@@ -20,33 +28,46 @@ class CustomVoiceControllerUI: UIViewController {
...
-                navigationViewController.modalPresentationStyle = .fullScreen
-                
-                strongSelf.present(navigationViewController, animated: true, completion: nil)
+                self?.presentNavigationWithCustomVoiceController(routeOptions, response)
             }
         }
     }
+
+    private func presentNavigationWithCustomVoiceController(_ routeOptions: NavigationRouteOptions,
+                                                            _ response: RouteResponse) {
```
Comment (1ec5) [inline Navigation-Examples/Examples/Custom-Voice-Controller.swift:37]: Is the `private` meaningful in the context of this method?

Arguments should be labeled. `_` is only for when there’s an inherent, self-explanatory order to the parameters. A better method signature would be `presentNavigation(response:)`. I think `RouteResponse.options` is the same as the original options object anyways, right?

File: Navigation-Examples/Examples/Custom-Voice-Controller.swift [hunk 2]
```diff
@@ -20,33 +28,46 @@ class CustomVoiceControllerUI: UIViewController {
...
+                                                        routeOptions: routeOptions,
+                                                        simulating: simulationIsEnabled ? .always : .onPoorGPS)
+
+        // `MultiplexedSpeechSynthesizer` will provide "a backup" functionality to cover cases, which
+        // our custom implementation cannot handle.
+        let speechSynthesizer = MultiplexedSpeechSynthesizer([CustomVoiceController(), SystemSpeechSynthesizer()])
+
+        // Create a `RouteVoiceController` type with a customized `SpeechSynthesizing` instance.
+        let routeVoiceController = RouteVoiceController(navigationService: navigationService,
+                                                        speechSynthesizer: speechSynthesizer)
```
Comment (1ec5) [inline Navigation-Examples/Examples/Custom-Voice-Controller.swift:50]: It’s helpful to provide line-level comments in these examples, but we should take the opportunity to say something beyond an English translation of the Swift code. What does RouteVoiceController do as part of the navigation experience?

File: Navigation-Examples/Examples/Custom-Voice-Controller.swift (hunk 1 above)
Comment (S2Ler) [reply Navigation-Examples/Examples/Custom-Voice-Controller.swift:37]: > Is the private meaningful in the context of this method?

removed. 

> I think RouteResponse.options is the same as the original options object anyways, right?

`RouteResponse.options`:

```swift
public enum ResponseOptions {
    case route(RouteOptions)
    case match(MatchOptions)
}
```

it is not strictly the same, but you can probably assume that `ResponseOptions` is `route(RouteOptions)` although it is not clear on API level.

File: Navigation-Examples/Examples/Custom-Voice-Controller.swift (hunk 2 above)
Comment (S2Ler) [reply Navigation-Examples/Examples/Custom-Voice-Controller.swift]: Added, please check.

**Review by MaximAlien — APPROVED** [review]

**Pull request merged**